src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -181,5 +183,28 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            if (Rows.Count() == 0)
+            {
+                throw new InvalidDataContractException("It's not possible to serialize an empty table.");
```

Comment:
>hrow new InvalidDataContractException("It's not possible to serialize an empty table."); [](http://example.com/codeflow?start=17&length=88)

We should support it

Reply:
Definitely - this was a core thing for Cards.
The challenge is that we don't have a good way to do it today  since we don't have type literals, and Table() is ambiguous. 

This is the same problem Blank() had. 
Need to brainstorm ....